### ISSUE TITLE
(URGENT) Unable to fetch ECMA-262

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-ecma#9]
```
[relaton-ecma] ("ECMA 262") fetching...
[relaton] ERROR: ECMA 262 -- undefined method `min' for false:FalseClass

        row = search(code).min { |a, b| compare_edition_volume a, b }
                          ^^^^
```

**Comment by andrew2net** [comment]
We should use "ECMA-262" not "ECMA 262", but it shouldn't fails, it should return "not found". I'll fix it.

**Comment by andrew2net** [comment]
Fixed in v1.14.4
```
$ relaton fetch "ECMA 262"
[relaton-ecma] ("ECMA 262") fetching...
[relaton-ecma] WARNING no match found online for ECMA 262. The code must be exactly like it is on the standards website.
No matching bibliographic entry found

$ relaton fetch "ECMA-262"
[relaton-ecma] ("ECMA-262") fetching...
[relaton-ecma] ("ECMA-262") found ECMA-262
<bibdata type="standard" schema-version="v1.2.3">
  <fetched>2023-06-06</fetched>
  <title format="text/plain" language="en" script="Latn">ECMAScript® 2022 language specification</title>
  <uri type="src">https://262.ecma-international.org/9.0/index.html</uri>
  <docidentifier type="ECMA" primary="true">ECMA-262</docidentifier>
  ...
```

**Comment by ronaldtse** [comment]
Let's support both "ECMA 262" and "ECMA-262". The distinction is not too clear for the user. Thanks!

**Comment by andrew2net** [comment]
Fixed in v 1.14.5
```
$ relaton fetch "ECMA-262"
[relaton-ecma] ("ECMA-262") fetching...
[relaton-ecma] ("ECMA-262") found ECMA-262
...

$ relaton fetch "ECMA 262"
[relaton-ecma] ("ECMA 262") fetching...
[relaton-ecma] ("ECMA 262") found ECMA-262
...
```

**Comment by ronaldtse** [comment]
Thank you @andrew2net !
